src/components/pages/rns/buy/DomainOffersCheckoutPage.tsx

```diff
@@ -116,6 +114,12 @@ const DomainOffersCheckoutPage: FC<{}> = () => {
   // check funds
   useEffect(() => {
     if (web3 && account && tokenId && !isFundsConfirmed) {
+      const {
+        item: {
+          paymentToken,
+        },
+      } = order
+      const currency = crypto[paymentToken]
```

Comment:
```suggestion
      const { symbol } = crypto[paymentToken]
```
If `symbol` is the only prop needed from currency